src/puppeteer_pool.js

```diff
@@ -20,6 +26,24 @@ const DEFAULT_OPTIONS = {
 
     // TODO: use settingsRotator()
     launchPuppeteerFunction: launchPuppeteerOptions => launchPuppeteer(launchPuppeteerOptions),
+
+    recycleDiskCache: false,
+};
+
+const mkdtempAsync = Promise.promisify(fs.mkdtemp);
+const rimrafAsync = Promise.promisify(rimraf);
+const DISK_CACHE_DIR = path.join(os.tmpdir(), 'puppeteer_disk_cache-');
+
+/**
+ * Deletes Chrome's user data directory
+ * @param diskCacheDir
+ */
+const deleteDiskCacheDir = (diskCacheDir) => {
+    log.debug('PuppeteerPool: Deleting disk cache directory', { diskCacheDir });
+    rimrafAsync(diskCacheDir)
```

Comment:
Missing return here? (I see that you chain it somewhere below)

Reply:
Fixed